src/main/java/model/Entities/Customer.java

```diff
@@ -1,4 +1,12 @@
 package model.Entities;
 
 public class Customer {
+    //Customer cellphone must be unique
+    long cellphone;
```

Comment:
don't really need a long here, int should be fine. cellphone number is only 10 digits

Reply:
I think the largest value for an int is 2147483647, but since theres no guarantee that the area code is less than 214, we still have to use a long

Reply:
ah, I must have counted wrong! Good catch!